Readme.md

```diff
@@ -84,44 +84,63 @@ the specifics for *pino-multi-stream*:
 
 [pinoapi]: https://github.com/pinojs/pino#api
 
-<a id="caveats"></a>
-## Caveats
+### pinoms.multistream(streams)
+
+Manually create a single `multistream` as used internally by the
+wrapper:
+
+```js
+var fs = require('fs')
+var pino = require('pino')
+var multistream = require('pino-multi-stream').multistream
```

Comment:
is it because we don't want to add props to function because function-object will perform worse?